src/textual/widgets/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+from ._footer import Footer as Footer
```

Comment:
According to [PEP 484](https://peps.python.org/pep-0484/#stub-files):
> Modules and variables imported into the stub are not considered exported from the stub unless the import uses the `import ... as ...` form or the equivalent `from ... import ... as ...` form.
> To clarify, the intention here is that only names imported using the form `X as X` will be exported, i.e. the name before and after as must be the same.

Reply:
Love the typing magic here. It's a little burden to maintain, but worth it.

Suggest putting a comment in `__init__.py` to remind people to update the `.pyi` when adding a new widget.

Reply:
I agree, and that's what I was trying to do with this comment there: :slightly_smiling_face: 
![Screenshot from 2022-06-17 12-14-59](https://user-images.githubusercontent.com/722388/174288037-4409e3c4-d12f-4277-9016-723b149f9fcd.png)
Are you thinking of another type of comment?

Reply:
That’s cool. Missed that!